wallet/src/util/WalletBackendAdapter.ts

```diff
@@ -321,15 +336,19 @@ export const makeWalletBridgeFromFollowers = (
             value.liveOffers ?? [],
           );
         },
+        err => {
+          console.error('error watching pending offers', err);
+          errorHandler(new Error(err));
```

Comment:
is this because `err` isn't necessarily an Error?

Reply:
Yea it's just a string, which I thought would be more convenient but not in this case